src/lib/rng/jitter_rng/jitter_rng.h

```diff
@@ -0,0 +1,48 @@
+/*
+* CPU Jitter Random Number Generator
+* (C) 2024 Planck Security S.A.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#ifndef BOTAN_JITTER_RNG_H_
+#define BOTAN_JITTER_RNG_H_
+
+#include <botan/rng.h>
+
+struct rand_data;
+
+namespace Botan {
+
+class BOTAN_DLL Jitter_RNG_Data {
```

Comment:
This should just be forward declared here, and defined in the source file

Reply:
A naked pointer to `Jitter_RNG_Data` works.